### ISSUE TITLE
overview: use `ct-form-layout` instead of styled `td`s for the labels

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#9773]
As mentioned in #9772:

The Overview page should use `<th>` for the labels in the table instead of using styled up `<td>`s. It could end up looking identical — however, this is important for accessibility reasons.

**Comment by garrett** [comment]
I thought I already sent in a PR for this, but I guess I might've had a local WIP branch… or dreamt I changed it. 
:zzz::man_shrugging::thought_balloon:

**Comment by martinpitt** [comment]
current System page still uses `<td>`.

**Comment by garrett** [comment]
Yep. I tried porting it earlier, but it's all interconnected and makes a bunch of assumptions in the HTML, CSS, and JavaScript.

I've been trying again today, and it's still not as easy as it seems. A naïve approach has all sorts of alignment issues.

Best solution would be to port to ct-forms-layout for the table and use grid to place the table and charts (and notifications warnings at top, when necessary).

In a WIP local solution, I tried doing this by overriding the layout for the structure, then override the th in the info table, but it's still not right. I think I'd have to use `!important` and it gets messier and messier with overrides on overrides and workarounds on workarounds. And then the sub-pages may be affected too. This is why it's not fixed yet, even though it _seems_ simple.

**Comment by garrett** [comment]
…And with the amount of effort porting everything over, we might as well have a newer, better layout design implemented while we're at it, since it would be the same amount of dev effort to properly fix it.

**Comment by garrett** [comment]
I think I should probably shelve the quick work-in-progress proof-of-concept I played around with today: https://github.com/garrett/cockpit/tree/system-overview-relayout

The differences are @ https://github.com/cockpit-project/cockpit/compare/master...garrett:system-overview-relayout?expand=1

Here's a screenshot:

![system overview hack](https://user-images.githubusercontent.com/10246/51107176-522f5600-17ee-11e9-9217-919c6ed15e22.png)

As you can see, the labels are not aligned properly still. I could fix it, but then I noticed that it affects other pages (and dialogs) and that the dialogs themselves are not correct either.

...So it's basically opening cans of worms all over the place.

It should be fixed, but it should be fixed correctly.
